src/db/connection.js

```diff
@@ -27,4 +27,12 @@ const closeDatabase = async (drop = false) => {
   await mongoose.connection.close();
 };
 
-module.exports = { connectToMongo, closeDatabase };
+const clearDatabase = async () => {
+  const { collections } = mongoose.connection;
+  /* eslint-disable no-restricted-syntax, guard-for-in, no-await-in-loop */
+  for (const key in collections) {
+    await mongoose.connection.dropCollection(key);
```

Comment:
It's usually not good practice to use await inside a for loop, because that means that next iteration won't start until previous iteration completes. But that does not make sense if each iteration is independent, they should run in parallel.

Can you checkout [this example](https://eslint.org/docs/rules/no-await-in-loop)? It shows how you can use await only once at the end of the for loop.